src/rpc/blockchain.cpp

```diff
@@ -2425,3 +2426,87 @@ UniValue getburnreport(const UniValue& params, bool fHelp)
 
     return json;
 }
+
+UniValue createmrcrequest(const UniValue& params, const bool fHelp) {
+    if (fHelp || params.size() > 2) {
+        throw runtime_error("createmrcrequest <force> <dry_run>\n"
+                            "\n"
+                            "<force> - If true, create the request even if it results "
+                            "in a reward loss or ban from the network. Defaults to false.\n"
+                            "\n"
+                            "<dry_run> - If true, calculate the reward and fee but do not "
+                            "send the contract. Defaults to false.\n"
+                            "\n"
+                            "Creates an MRC request. Requires an unlocked wallet.");
+    }
```

Comment:
These don't appear to be required arguments, so they should follow the convention for optional arguments
```suggestion
        throw runtime_error("createmrcrequest [force [dry_run]]\n"
                            "\n"
                            "[force] - If true, create the request even if it results "
                            "in a reward loss or ban from the network. Defaults to false.\n"
                            "\n"
                            "[dry_run] - If true, calculate the reward and fee but do not "
                            "send the contract. Defaults to false.\n"
                            "\n"
                            "Creates an MRC request. Requires an unlocked wallet.");
    }
```

Reply:
I agree but I will do it on the mrc branch and flow it through.